ext/openssl/ossl_pkey_ec.c

```diff
@@ -1232,10 +1235,13 @@ static VALUE ossl_ec_point_eql(VALUE a, VALUE b)
     GetECPoint(b, point2);
     GetECGroup(group_v1, group);
 
-    if (EC_POINT_cmp(group, point1, point2, ossl_bn_ctx) == 1)
-        return Qfalse;
+    switch (EC_POINT_cmp(group, point1, point2, ossl_bn_ctx)) {
+    case 0: return Qtrue;
+    case 1: return Qfalse;
+    default: ossl_raise(cEC_POINT, "EC_POINT_cmp");
```

Comment:
cEC_POINT
```suggestion
    default: ossl_raise(eEC_POINT, "EC_POINT_cmp");
```

Reply:
It looks like there are several other calls with this same error, where ossl_raise() is called with cEC_POINT. Should I update those as well?